pkg/apis/core/validation/validation.go

```diff
@@ -3165,6 +3167,12 @@ func validateContainerCommon(ctr *core.Container, volumes map[string]core.Volume
 		allErrs = append(allErrs, field.Required(path.Child("image"), ""))
 	}
 
+	if len(ctr.Image) > 0 {
+		_, err := dockerref.ParseNormalizedNamed(ctr.Image)
+		if err != nil {
+			allErrs = append(allErrs, field.Invalid(path.Child("image"), ctr.Image, err.Error()))
```

Comment:
Rather than cloning this vendor package, I am just using the error message.  

If repository name for docker is all capitals:

```
The Job "capitals-no-failure" is invalid: spec.template.spec.containers[0].image: Invalid value: "BUSYBOX": invalid reference format: repository name must be lowercase
```

Reply:
Changed this so we follow similar error message as other cases in validation.